submission_runner.py

```diff
@@ -441,4 +472,5 @@ def main(_):
 
 
 if __name__ == '__main__':
+  flags.mark_flag_as_required('experiment_dir')
```

Comment:
we should also mark the following as required:
workload
framework
submission_path
tuning_ruleset
tuning_search_space